pkg/operation/botanist/vpnseedserver.go

```diff
@@ -80,7 +80,7 @@ func (b *Botanist) DefaultVPNSeedServer() (vpnseedserver.Interface, error) {
 			ServiceCIDR: b.Shoot.Networks.Services.String(),
 			NodeCIDR:    pointer.StringDeref(b.Shoot.GetInfo().Spec.Networking.Nodes, ""),
 		},
-		Replicas:                             1,
+		Replicas:                             b.Shoot.GetReplicas(1),
```

Comment:
Is it possible to add unit test to prevent regressing the case in the future?

Reply:
A unit test has been added. Additionally the e2e test for the hibernated shoot has been extended to check for running pods in the control plane in hibernated state.